### PR TITLE
Hotfix/er 948 cleanup frontend compiling

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/ER-948

New frontend compiling.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 COMPOSE_PROJECT_NAME=ereolen
 COMPOSE_DOMAIN=ereolen.local.itkdev.dk
-REMOTE_HOST=ereolen-p02.dbc.dk
-REMOTE_DB_DUMP_CMD='drush --root=/data/www/ereolen_dk/htdocs --uri=ereolen.dk sql-dump'
-REMOTE_PATH='/data/www/ereolen_dk/htdocs/sites/default/files'
+REMOTE_HOST=aarhus-s02.dbc.dk
+REMOTE_DB_DUMP_CMD='drush --root=/data/www/stg_ereolen_dk/htdocs --uri=stg.ereolen.dk sql-dump'
+REMOTE_PATH='/data/www/stg_ereolen_dk/htdocs/sites/default/files'
```
Comment (rimi-itk) [inline .env:5]: I'm not sure we should change this – or commit it in the first place, but that's another issue.

File: README.md [hunk 2]
```diff
@@ -157,13 +157,13 @@ docker-compose run --rm drush [command]
 # Building themes
 
 ```sh
-docker-compose run --rm node bash -c "cd /app/sites/all/themes/orwell/ && npm install"
-docker-compose run --rm node bash -c "cd /app/sites/all/themes/orwell/ && node_modules/.bin/gulp sass"
+docker-compose run --rm node bash -c "cd /app/sites/all/themes/orwell/ && yarn install"
+docker-compose run --rm node bash -c "cd /app/sites/all/themes/orwell/ && yarn dev"
```
Comment (rimi-itk) [inline README.md:161]: This should probably be `… && yarn build`.

File: README.md [hunk 3]
```diff
@@ -157,13 +157,13 @@ docker-compose run --rm drush [command]
...
 ```
 
 ```sh
-docker-compose run --rm node bash -c "cd /app/sites/all/themes/wille/ && npm install"
-docker-compose run --rm node bash -c "cd /app/sites/all/themes/wille/ && node_modules/.bin/gulp sass"
+docker-compose run --rm node bash -c "cd /app/sites/all/themes/wille/ && yarn install"
+docker-compose run --rm node bash -c "cd /app/sites/all/themes/wille/ && yarn dev"
```
Comment (rimi-itk) [inline README.md:166]: This should probably be `… && yarn build`.

File: sites/all/themes/orwell/orwell.info [hunk 4]
```diff
@@ -3,15 +3,15 @@ name = Orwell
...
-stylesheets[all][] = css/orwell.main.css
 stylesheets[all][] = system.menus.css
 stylesheets[all][] = system.theme.css
 stylesheets[all][] = field.css
 stylesheets[all][] = twocol_stacked.css
 stylesheets[all][] = search.css
 stylesheets[all][] = panels.css
 ; Kill ding_carousels theming (we got our own).
-stylesheets[all][] = ding_carousel.css
+stylesheets[all][] = build/app.css
```
Comment (rimi-itk) [inline sites/all/themes/orwell/orwell.info:13]: Do we still kill what needs to be killed?

File: sites/all/themes/wille/assets/sass/base/_vars.scss [hunk 5]
```diff
@@ -1,6 +1,10 @@
 /**
  * Colors.
  */
+
+@import '~bourbon';
+$font_dir_path : '/assets/fonts';
```
Comment (rimi-itk) [inline sites/all/themes/wille/assets/sass/base/_vars.scss:6]: Is `$font_dir_path` used anywhere?

File: sites/all/themes/wille/assets/sass/base/_vars.scss [hunk 6]
```diff
@@ -52,13 +56,58 @@ $title-color: $black-medium;
...
-@include font-face("OpenSans Regular Italic", "../fonts/OpenSans-RegularItalic", $file-formats: woff ttf);
-@include font-face("OpenSans Bold", "../fonts/OpenSans-Bold", $file-formats: woff ttf);
-@include font-face("OpenSans Bold Italic", "../fonts/OpenSans-BoldItalic", $file-formats: woff ttf);
+@include font-face("OpenSans Light", #{$font_dir_path}/OpenSans-Light, $file-formats: woff);
+@include font-face("OpenSans Light Italic", #{$font_dir_path}/OpenSans-LightItalic, $file-formats: woff);
+@include font-face("OpenSans Regular", #{$font_dir_path}/OpenSans-Regular, $file-formats: woff);
+@include font-face("OpenSans Regular Italic", #{$font_dir_path}/OpenSans-RegularItalic, $file-formats: woff);
+@include font-face("OpenSans Bold", #{$font_dir_path}/OpenSans-Bold, $file-formats: woff);
+@include font-face("OpenSans Bold Italic", #{$font_dir_path}/OpenSans-BoldItalic, $file-formats: woff);
+*/
```
Comment (rimi-itk) [inline sites/all/themes/wille/assets/sass/base/_vars.scss:110]: Remove code that's not needed.

File: sites/all/themes/wille/assets/sass/base/_vars.scss [hunk 7]
```diff
@@ -69,10 +118,10 @@ $font-bold: 'OpenSans Bold', $fallbackFontFamily;
 $font-bold-italic: 'OpenSans Bold Italic', $fallbackFontFamily;
 
 // Icon font.
-@include font-face("icomoon", "../fonts/icomoon", $file-formats: ttf woff svg eot);
+// @include font-face("icomoon", #{$font_dir_path}/icomoon, $file-formats: woff);
 
 // Mikado font.
-@include font-face("Mikado", "../fonts/mikado", $file-formats: eot woff woff2 ttf);
+//@include font-face("Mikado", #{$font_dir_path}/mikado, $file-formats: woff);
```
Comment (rimi-itk) [inline sites/all/themes/wille/assets/sass/base/_vars.scss:124]: Remove if not needed.

File: sites/all/themes/wille/assets/sass/wille.main.scss [hunk 8]
```diff
@@ -1,7 +1,8 @@
 // Base.
-@import 'bourbon';
+@import '~bourbon';
+@import '~bourbon-neat';
 @import 'base/functions';
-@import 'base/vars';
+//@import 'base/vars';
```
Comment (rimi-itk) [inline sites/all/themes/wille/assets/sass/wille.main.scss:5]: Remove if not needed.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
A large handful of comments added.

File: .env (hunk 1 above)
Comment (martinyde) [reply .env:5]: Well the dbc reference isn't right as that is no longer the prod server.

File: sites/all/themes/orwell/orwell.info (hunk 4 above)
Comment (martinyde) [reply sites/all/themes/orwell/orwell.info:13]: Ohh, yeah, is suppose we should

File: README.md (hunk 2 above)
Comment (martinyde) [reply README.md:161]: Changed

File: README.md (hunk 3 above)
Comment (martinyde) [reply README.md:166]: Changed

File: sites/all/themes/wille/assets/sass/base/_vars.scss (hunk 5 above)
Comment (martinyde) [reply sites/all/themes/wille/assets/sass/base/_vars.scss:6]: Removed

File: sites/all/themes/wille/assets/sass/base/_vars.scss (hunk 6 above)
Comment (martinyde) [reply sites/all/themes/wille/assets/sass/base/_vars.scss:110]: Removed

File: sites/all/themes/wille/assets/sass/base/_vars.scss (hunk 7 above)
Comment (martinyde) [reply sites/all/themes/wille/assets/sass/base/_vars.scss:124]: Removed

File: sites/all/themes/wille/assets/sass/wille.main.scss (hunk 8 above)
Comment (martinyde) [reply sites/all/themes/wille/assets/sass/wille.main.scss:5]: Removed

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**